src/components/DailyMailModal/DailyMailModal.service.ts

```diff
@@ -0,0 +1,27 @@
+import { TicketsOutboundDto } from "generated/model";
+
+export const getDailyPoints = (tickets: TicketsOutboundDto[]) => {
+    let lateBy = 0, toValidate = 0, pointsDone = 0
+
+
+    for (const ticket of tickets) {
+        switch (ticket.status) {
+            case "DONE":
+                pointsDone += ticket.ticketPoint ?? 0 // WHY NULLABLE
```

Comment:
It could be nullable in some providers and the optional case is not handled yet.

Some providers are saving this field as "___extra_fields_13123123123" and the plugin to collect the tickets is not super reliable yet and is in fact, an optional field when the ticket didn't went through estimation.